openapitools.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "./node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
+  "generator-cli": {
+    "version": "6.0.0",
+    "generators" : {
+      "python_legacy": {
+        "generatorName": "python-legacy",
+        "output": "./rest_sdk/python",
```

Comment:
@AliakseiButurliaa can you please use sdk-rest instead of rest_sdk for all occurences. Thanks.